Zend/zend_language_scanner.l

```diff
@@ -1494,13 +1494,6 @@ NEWLINE ("\r"|"\n"|"\r\n")
 	RETURN_TOKEN(T_DOUBLE_CAST);
 }
 
-<ST_IN_SCRIPTING>"("{TABS_AND_SPACES}"real"{TABS_AND_SPACES}")" {
-	if (PARSER_MODE()) {
-		zend_error(E_DEPRECATED, "The (real) cast is deprecated, use (float) instead");
```

Comment:
I'd prefer to keep lexer support and throw an error here. That way ext/tokenizer can still be used on old code (e.g. to migrate real cast to float cast...)

Reply:
Makes sense, what type of error should I use? ``E_PARSE`` ?

Reply:
`zend_throw_exception(zend_ce_parse_error`